next/components/forms/segments/UserProfile/UserProfileDetailsButtons.tsx

```diff
@@ -17,44 +17,48 @@ const UserProfileDetailsButtons = (props: UserProfileDetailsButtonsProps) => {
 
   return (
     <div className="width-fit">
-      {isEditing ? (
-        <div className="flex flex-row gap-5 items-center">
-          <Button
-            className={cx('hidden focus:bg-white h-full', 'md:block')}
-            variant="plain-black"
-            size="sm"
-            text={t('profile_detail.stop_edit_button')}
-            onPress={onCancelEditing}
-          />
-          <Button
-            className={cx('hidden', 'md:block')}
-            variant="black"
-            size="sm"
-            text={t('profile_detail.save_edit_button')}
-            type="submit"
-            form={formId}
-          />
-          <CloseIcon
-            className={cx('block cursor-pointer w-6 h-6', 'md:hidden')}
-            onClick={onCancelEditing}
-          />
-        </div>
-      ) : (
-        <div className="width-fit">
-          <Button
-            variant="black"
-            startIcon={<WhiteEditIcon fill="white" className="w-6 h-6" />}
-            size="sm"
-            text={t('profile_detail.start_edit_button')}
-            className="hidden md:block"
-            onPress={() => onChangeIsEditing(true)}
-          />
-          <WhiteEditIcon
-            className="block md:hidden cursor-pointer w-6 h-6"
-            onClick={() => onChangeIsEditing(true)}
-          />
-        </div>
-      )}
+      {
+        // first button is to fix bug with autofocus of button 'cancel edit'
```

Comment:
I rather think you can do that differently, i.e. by autofocusing the first input field, rather than focusing an invisible button (I think that would come with it's own issues)

Reply:
@mpinter no, autofocu is not working. Of course I tried it. I tried lot of things for few hours. It's ignoring everthing and always focusing on first button.

Reply:
ok, lgtm, will add it as a backlog task for later (there has to be a better way)